packages/editor/src/lib/editor/Editor.ts

```diff
@@ -2451,13 +2450,13 @@ export class Editor extends EventEmitter<TLEventMap> {
 	 * editor.setCamera(editor.getCamera())
 	 * ```
 	 *
-	 * @param options - The camera options to set.
+	 * @param opts - The camera options to set.
```

Comment:
Renamed options to opts, so that we now consistently use opts (at least in the Editor, we might have other places to update still).